docs/getting_started.md

```diff
@@ -1,9 +1,33 @@
 Getting Started
 ================
-1. Run the `setup.sh` script to find & replace placeholders with information about your project
-2. Fill out `README.md` with details about your project
-3. Edit documentation configuration in `conf.py`
-    - For the full list of available options, see [the official documentation][Sphinx-docs].
-4. Rewrite this page with instructions for how to get started with your project
+
+### Setup
+1. Ensure batteries are charged
+2. Connect flight computers via USB cable
+3. Start AltOS
+4. Download and delete all on-board flight logs
+5. Configure conditions for recovery and staging events
```

Comment:
Please add links to the AltOS documentation for these steps as these instructions aren't immediately clear to anyone not already familiar with the configuration process.

Reply:
see downloads